src/gt4py/next/config.py

```diff
@@ -73,6 +73,9 @@ def env_flag_to_bool(name: str, default: bool) -> bool:
 )
 
 
+GTFN_SOURCE_CACHE_DIR: str = os.environ.get(f"{_PREFIX}_GTFN_SOURCE_CACHE_DIR", "gtfn_cache")
```

Comment:
I would prefer to avoid exposing a config option that is special to a certain backend. Is it necessary?

Reply:
We introduced this since we wanted to avoid hardcoding it. In almost all cases, this should not change but e.g. in the tests, where the cache should be deleted afterwards, specifying a different directory would make sense.
Do you have a suggestion on how to make this configurable which you would prefer?

Reply:
DaCe has also a cache directory, maybe we could use a common config option for compiled backends?

Reply:
Yes, that sounds good to me. Do you agree @havogt?
And do you have a name in mind?

Reply:
I originally proposed this, but after thinking about it again, and together with my other suggestion about how to deal with this in tests, I think it's not needed at least for this PR.

Reply:
Ok, I removed it and hardcoded it in gtfn.py.